code/modules/hydroponics/seed.dm

```diff
@@ -737,11 +737,11 @@
 		if(GENE_BIOCHEMISTRY)
 			P.values["[TRAIT_CHEMS]"] =        chems
 			P.values["[TRAIT_EXUDE_GASSES]"] = exude_gasses
-			traits_to_copy = list(TRAIT_POTENCY, TRAIT_SPORING, TRAIT_BENEFICIAL_REAG, TRAIT_MUTAGENIC_REAG, TRAIT_TOXIC_REAG)
+			traits_to_copy = list(TRAIT_POTENCY, TRAIT_BENEFICIAL_REAG, TRAIT_MUTAGENIC_REAG, TRAIT_TOXIC_REAG)
 		if(GENE_OUTPUT)
 			traits_to_copy = list(TRAIT_PRODUCES_POWER,TRAIT_BIOLUM)
 		if(GENE_ATMOSPHERE)
-			traits_to_copy = list(TRAIT_HEAT_TOLERANCE,TRAIT_LOWKPA_TOLERANCE,TRAIT_HIGHKPA_TOLERANCE)
+			traits_to_copy = list(TRAIT_HEAT_TOLERANCE,TRAIT_LOWKPA_TOLERANCE,TRAIT_HIGHKPA_TOLERANCE,TRAIT_SPORING)
```

Comment:
I don't think it makes much sense to move Sporing to Atmosphere (which is mostly plant's ability to endure atmosphere, not its interactions with it). Exhuding gasses is under Biochemistry and unique production features are under Output and Metabolism. It makes more sense under any of those three than it does there.

Reply:
I can move it to Output instead -- that one has so few results as it is that it'd be nice to give it some more love. I don't like keeping it biochemistry, because it strictly limits it to luck of the draw to get a non toxic spore plant if you want to do neat things with it, mostly owing to how the plant hardware works. At some point, I'd like to make the bioballistic able to target certain subsectors of genes, but that is a very long ways out.

Reply:
That'd be nice as long as its in gene that, well, makes sense for it. Honestly, just more consistency in categories sake rather than balance.